notebooks/en/llm_judge.ipynb

```diff
@@ -0,0 +1,744 @@
+{
```

Comment:
<p>missing a word here: "But we'll see that it will not well out-of-the-box: you need to set it up carefully for good results." =&gt; ?  "But we'll see that it will not <strong>work</strong> well out-of-the-box: you need to set it up carefully for good results."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Done!